### DISCUSSION TITLE
Real-time Validation $this->validateOnly() not working for rule[ input => lte:name.*.name]

### DISCUSSION THREAD

**Discussion by BilawalDeveloper** [livewire/livewire#4163] [7. Bugs (archived)]
`protected $rules = [
        'subjectsResult.*.obtainedMarks' => 'lte:subjectsResult.*.totalMarks',
    ];`

**Not detecting second input fields 'lte:subjectsResult.*.totalMarks' **
First Input group:
`<input wire:model.delay="subjectsResult.{{ "subject_". $subject->id }}.totalMarks" type="text"
                           id="student-result-total-marks-{{ $subject->id }}"
                           onchange="setSubjectId(this)"
                           data-input-field="subjectsResult.{{ "subject_". $subject->id }}.subjectId"
                           data-input-value="{{ $subject->id }}"
                           class="form-control @error("subjectsResult.subject_$subject->id.totalMarks") is-invalid @enderror"
                           placeholder="(required*)">`

Second input group:
    ` <input wire:model.delay="subjectsResult.{{ "subject_". $subject->id }}.obtainedMarks"
                 type="text" id="student-result-obtained-marks-{{ $subject->id }}"
                 class="form-control @error("subjectsResult.subject_$subject->id.obtainedMarks") is-invalid @enderror"
                 placeholder="(required*)">`

**Comment by ralphjsmit** [comment]
Hey Bilawal, my guess is that because you have the asterisk `*` in your validation rule (`lte:subjectsResult.*.totalMarks`), Livewire doesn't know which field to take and compare against. There may be multiple options.

Consider this:
```
<input wire:model.delay="subjectsResult.1.totalMarks">
<input wire:model.delay="subjectsResult.2.totalMarks">
```
Should `subjectsResult.1.obtainedMarks` map to `subjectsResult.1.totalMarks` or `subjectsResult.2.totalMarks`?

For humans that's obvious, but for Livewire it unfortunately isn't.

Can you share how your `$this->validateOnly()` looks like and which properties you have on your Livewire component? Perhaps we can make a workaround for this.
